lib/modules/dosomething/dosomething_rogue/dosomething_rogue.cron.inc

```diff
@@ -29,33 +29,37 @@
           $mime_split = explode(':', $task->file);
           $task->file = $mime_split[0] . ':' . $mimetype . $mime_split[1];
         }
-        $values = [
+
+        $data = [
           'nid' => $task->campaign_id,
```

Comment:
I was getting a little caught up on this because in order to send it to `dosomething_rogue_transform_reportback`, we'd be sending in only the `$task` object to fill out `$data` array and seemed to not match up exactly so thought it was cleaner just to keep as is?